src/GridFieldVersionedOrderableRows.php

```diff
@@ -45,12 +53,15 @@ protected function reorderItems($list, array $values, array $order)
                 );
             }
         }
-        
+
         $this->extend('onAfterReorderItems', $list);
-        
+
```

Comment:
Since you're changing this anyway, can you remove the newline before the closing curly brace?